io.catenax.generic.digital_product_passport/2.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,1142 @@
+
+:LifeTypeEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Life Type Enumeration"@en ;
+   samm:description "Enumeration with the values guaranteed lifetime, technical lifetime and mean time between failures."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "guaranteed lifetime" "technical lifetime" "mean time between failures" ) .
+
+:LinearCharacteristic a samm:Characteristic ;
+   samm:preferredName "Linear Characteristic"@en ;
+   samm:description "Characteristic for linear measurements of an item."@en ;
+   samm:dataType :LinearEntity .
+
+:MassCharacteristic a samm:Characteristic ;
+   samm:preferredName "Mass Characteristic"@en ;
+   samm:description "Characteristic for mass measurements of an item."@en ;
+   samm:dataType :MassEntity .
+
+:CapacityCharacteristic a samm:Characteristic ;
+   samm:preferredName "Capacity Characteristic"@en ;
+   samm:description "Characteristic for capacity measurements of an item."@en ;
+   samm:dataType :CapacityEntity .
+
+:PhysicalStateEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Physical State Enumeration"@en ;
+   samm:description "Enumeration with the values solid, liquid and gas."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "solid" "liquid" "gas" ) .
+
+:EoriTrait a samm-c:Trait ;
+   samm-c:baseCharacteristic :IdentifierCharacteristic ;
+   samm-c:constraint :EoriConstraint .
+
+:CodeEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Code Enumeration"@en ;
+   samm:description "Enumeration for GTIN, hash, DID, ISBN, TARIC or other."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "TARIC" "GTIN" "DID" "ISBN" "hash" "other" ) .
+
+:IdentifierEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Identifier Enumeration"@en ;
+   samm:description "Enumeration with \"PartInstanceID\" \"BatchID\" and \"ModelID\"."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "PartInstanceID" "BatchID" "ModelID" ) .
+
+:UnitCatalog a samm:Characteristic ;
+   samm:preferredName "Unit Catalog"@en ;
+   samm:description "Link to the unit catalog with all possible units in the format \"unit:xxx\"."@en ;
+   samm:see <https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/appendix/unitcatalog.html> ;
+   samm:dataType xsd:string .
+
+:DataTypeEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Data Type Enumeration"@en ;
+   samm:description "Data type that describe the content of the attributes children and data. In case \"object\" is selected in the enumeration, the children field will be used in the AdditionalDataEntity instead of the \"data\" property. If it is an other type, the content will be specified in \"data\" as a string."@en ;
+   samm:see <https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html> ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "array" "object" "xsd:string" "xsd:integer" "xsd:boolean" "xsd:double" "xsd:float" "xsd:byte" ) .
```

Comment:
if you have the xsd curie in the string, You need to expand this namespace somewhere to make clear, which version of XSD is used. you probably refer to the one used in this model (check namespace import of this model)

Reply:
@bs-jokri I don't quite undestand this, can you explain how?